app/modules/front-door/locals.tf

```diff
@@ -1,8 +1,11 @@
 locals {
+  service_name = "back-office-applications-docs"
```

Comment:
Is this meant to be at the module level? I'd expect this just in the stack maybe.